.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: If something isn't working as expected
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Make sure that you are submitting a new bug that was not previously reported or already fixed.
+
+        Please use a concise and distinct title for the issue.
+
+        If possible, attach screenshots or videos to illustrate the problem.
+  - type: textarea
+    attributes:
+      label: Detailed description
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps to reproduce the problem
+      description: What were you trying to do?
+      value: |
+        1.
+        2.
+        3.
+        ...
```

Comment:
Maybe add "expected result / actual result" ?

Reply:
Not sure. I personally find that very annoying whenever I see it. But if you think it helps I can add it.